XCUITests/NewTabSettings.swift

```diff
@@ -83,4 +84,41 @@ class NewTabSettingsTest: BaseTestCase {
         waitForExistence(app.textFields["url"])
         waitForValueContains(app.textFields["url"], value: "mozilla")
     }
+    
+    func testChangeNewTabSettingsLabel() {
+        //Go to New Tab settings and select Custom URL option
+        navigator.goto(NewTabSettings)
+        waitForExistence(app.navigationBars["New Tab"])
+        navigator.performAction(Action.SelectNewTabAsCustomURL)
+        //Enter a custom URL
+        app.textFields["NewTabAsCustomURLTextField"].typeText(websiteUrl)
+        app.textFields["NewTabAsCustomURLTextField"].typeText(XCUIKeyboardKey.return.rawValue)
```

Comment:
I think this line is not needed...as the typeText is done correctly in the previous one...